### PR TITLE
feat: support more zil eco projects

### PR DESCRIPTION
**Pull request by renlulu**

### CODE REVIEW

File: projects/zeeves/project.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (AshIgnYeo) [inline projects/zeeves/project.md:1]: zeeves seems to have formatting issues to the frontmatter. not displaying properly

File: projects/zilliqa-duel/project.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Zilliqa Duel
+description: False
+categories: false
```
Comment (AshIgnYeo) [inline projects/zilliqa-duel/project.md:4]: @AmritKumar should we include in those projects which have no information whatsoever? `false` for categories will currently break the script. I won't update the script at present unless this is a case that might happen for future projects?

**Review by AshIgnYeo — CHANGES_REQUESTED** [review]

File: projects/zhip/project.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+name: Zhip
+description: Zhip is the world's first iOS wallet for the Zilliqa blockchain
+categories: false
```
Comment (AmritKumar) [inline projects/zhip/project.md:4]:
```suggestion
categories: wallet,web3
```

File: projects/zhip/project.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+status: False
```
Comment (AmritKumar) [inline projects/zhip/project.md:5]:
```suggestion
status: Live
```

File: projects/zilliqa-duel/project.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+name: Zilliqa Duel
+description: False
```
Comment (AmritKumar) [inline projects/zilliqa-duel/project.md:3]:
```suggestion
description: dapp
```

File: projects/zilliqa-duel/project.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+categories: false
+status: False
```
Comment (AmritKumar) [inline projects/zilliqa-duel/project.md:5]:
```suggestion
status: Live
```

File: projects/zilliqa-duel/project.md (hunk 2 above)
Comment (AmritKumar) [reply projects/zilliqa-duel/project.md:4]: We have fixed the issue.

Comment (AshIgnYeo) [reply projects/zilliqa-duel/project.md:4]: @AmritKumar the current categories still show false though

**Review by AshIgnYeo — CHANGES_REQUESTED** [review]

File: projects/zilliqa-duel/project.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+name: Zilliqa Duel
+description: dapp
```
Comment (AmritKumar) [inline projects/zilliqa-duel/project.md:3]:
```suggestion
description: A gaming application.
```

File: projects/zilliqa-duel/project.md (hunk 2 above)
Comment (AmritKumar) [reply projects/zilliqa-duel/project.md:4]:
```suggestion
categories: dapp,gaming
```

**Review by AshIgnYeo — APPROVED** [review]

**Pull request merged**